tests/bgp/test_passive_peering.py

```diff
@@ -22,12 +22,12 @@
 
 
 @pytest.fixture(scope='module')
-def setup(tbinfo, nbrhosts, duthosts, rand_one_dut_hostname, request):
```

Comment:
Could you please share if this is a Day 1 failure or a regression issue? 
Seems rand_one_dut_hostname's behavior changed recently by https://github.com/sonic-net/sonic-mgmt/pull/12824

Reply:
Hi @lipxu , Thanks for reviewing. From our nightly T2 OC run results of June and July 2024, we could see the tests passing with the selection of line card in random all the time. Since August OC runs, we started seeing the failure above by selecting supervisor as dut in random.

Unfortunately, we don't have logs before June to compare.

Reply:
Hi @lipxu, I had a chance to test without the PR 12824 changes and just by using _rand_one_dut_hostname_ on the test as before. I could see even with that, supervisor gets picked up as DUT for testing.

![image](https://github.com/user-attachments/assets/2bb256c8-879f-4206-b87a-e9d062db34ed)

Reply:
Thanks for your information. Anyway, rand_one_dut_front_end_hostname would be more suitable.